test/e2e/container_iface_name_test.go

```diff
@@ -9,13 +9,10 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func isDebianRunc(pTest *PodmanTestIntegration) bool {
+// FIXME 2024-05-14: "Debian" here is a proxy for "netavark < 1.10"
```

Comment:
we could parse podman info to get the real version

At this point I seriously question if we should continue using the debian sid distro versions.
Netavark v1.5 has been released on Feb 2, 2023, yet SID is still on 1.4 somehow. Not for this PR of course but I really wonder if we have to consider alternatives.

Reply:
Thank you for bringing this up, that's the broader question we need to address next. The entire purpose of testing in Debian was quick discovery of problems that would affect RHEL. This may be a good time to evaluate what testing gaps we have and how we can fill them.

Reply:
As we turn on zstd:chunked, composefs, and others for Fedora, we could continue with old checks for Debian.